pkg/controller/config-controller.go

```diff
@@ -156,6 +162,47 @@ func (r *CDIConfigReconciler) reconcileStorageClass(config *cdiv1.CDIConfig) err
 	return nil
 }
 
+func (r *CDIConfigReconciler) reconcileDefaultPodResourceRequirements(config *cdiv1.CDIConfig) error {
+	log := r.Log.WithName("CDIconfig").WithName("DefaultPodResourceRequirements")
+
+	// TODO: set as default value (0)
+	log.Info("Setting default pod resource requirements as default value (0)")
+	config.Status.DefaultPodResourceRequirements = &v1.ResourceRequirements{
+		Limits: map[v1.ResourceName]resource.Quantity{
+			v1.ResourceCPU:    *resource.NewQuantity(0, resource.DecimalSI),
+			v1.ResourceMemory: *resource.NewQuantity(0, resource.DecimalSI)},
+		Requests: map[v1.ResourceName]resource.Quantity{
+			v1.ResourceCPU:    *resource.NewQuantity(0, resource.DecimalSI),
+			v1.ResourceMemory: *resource.NewQuantity(0, resource.DecimalSI)},
+	}
+
+	if config.Spec.PodResourceRequirements != nil {
+		// TODO: Override
```

Comment:
I think this TODO is done, I also think the log is not needed as its normal operation.

Reply:
Done!